Bot/Commands/DND/AgreeToTerms.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Discord;
+using Discord.WebSocket;
+using NetBot.Bot.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NetBot.Bot.Commands.DND
+{
+    public class AgreeToTerms : ISlashCommand
+    {
+        public SlashCommandBuilder SlashCommandBuilder => new SlashCommandBuilder()
+            .WithName("agreetoterms")
+            .WithDescription("List the terms associated with using DND related commands, and choose to either accept or deny them");
+
+        public ulong? Guild => null;
+
+        public async Task CommandEvent(SocketSlashCommand command)
+        {
+            EmbedBuilder embed = new EmbedBuilder()
```

Comment:
This part isn't really related to this PR specifically, but in the future we should try to handle this part outside of the command classes